jvm/src/main/scala/tastyquery/jdk/ClasspathSources.scala

```diff
@@ -0,0 +1,22 @@
+package tastyquery.jdk
+
+import java.io.File
+
+object ClasspathSources {
+
+  def classloaderURLs: IArray[String] = {
+    import scala.language.unsafeNulls
+    val cl = ClassLoader.getSystemClassLoader
+    val urls = cl.asInstanceOf[java.net.URLClassLoader].getURLs
+    val files = urls.map(_.getFile)
+    IArray.unsafeFromArray(files)
```

Comment:
I doubt the performance requirements of this method warrant the usage of unsafe API.

Reply:
I am using IArray as the main collection type throughout, I didn't think this api wasn't recommended, just that it should not be used if you do not know how the underlying array could be shared.

Reply:
It's better to use `files.toIArray` (if that exists) or `IArray.from(files)` when the path is not performance-critical.